docs/examples/junit4/generic/src/test/java/generic/CmdModifierTest.java

```diff
@@ -39,7 +41,17 @@ public void testHostnameModified() throws IOException, InterruptedException {
 
     @Test
     public void testMemoryLimitModified() throws IOException, InterruptedException {
-        final Container.ExecResult execResult = memoryLimitedRedis.execInContainer("cat", "/sys/fs/cgroup/memory/memory.limit_in_bytes");
+        final Container.ExecResult execResult = memoryLimitedRedis.execInContainer("cat", getMemoryLimitFilePath());
         assertEquals(String.valueOf(memoryInBytes), execResult.getStdout().trim());
     }
+
+    private String getMemoryLimitFilePath() {
+        Info info = memoryLimitedRedis.getDockerClient().infoCmd().exec();
```

Comment:
I'm not sure that we should encourage `getDockerClient()` to be used to obtain a client - I think this is unintentionally part of the public API.

Please could we use `DockerClientFactory.instance().client()` to obtain a client instead?

Reply:
Sure, I replaced the `getDockerClient` call with `DockerClientFactory.instance().client()`

Reply:
Thanks - no worries about rebasing/updating the branch; we try to reduce the amount of CI time burned on that kind of thing when possible.